### ISSUE TITLE
Missing throws clauses

### ISSUE THREAD

**Issue by elharo** [googleapis/google-oauth-java-client#598]
```
[WARN] D:\a\google-oauth-java-client\google-oauth-java-client\google-oauth-client-java6\src\main\java\com\google\api\client\extensions\java6\auth\oauth2\AuthorizationCodeInstalledApp.java:47:43: Expected @throws tag for 'IOException'. [JavadocMethod]
[WARN] D:\a\google-oauth-java-client\google-oauth-java-client\google-oauth-client-java6\src\main\java\com\google\api\client\extensions\java6\auth\oauth2\AuthorizationCodeInstalledApp.java:98:53: Expected @throws tag for 'IOException'. [JavadocMethod]
[WARN] D:\a\google-oauth-java-client\google-oauth-java-client\google-oauth-client-java6\src\main\java\com\google\api\client\extensions\java6\auth\oauth2\FileCredentialStore.java:69:73: Expected @throws tag for 'IOException'. [JavadocMethod]
[WARN] D:\a\google-oauth-java-client\google-oauth-java-client\google-oauth-client-java6\src\main\java\com\google\api\client\extensions\java6\auth\oauth2\FileCredentialStore.java:106:54: Expected @throws tag for 'IOException'. [JavadocMethod]
[WARN] D:\a\google-oauth-java-client\google-oauth-java-client\google-oauth-client-java6\src\main\java\com\google\api\client\extensions\java6\auth\oauth2\FileCredentialStore.java:187:77: Expected @throws tag for 'IOException'. [JavadocMethod]
[WARN] D:\a\google-oauth-java-client\google-oauth-java-client\google-oauth-client-java6\src\main\java\com\google\api\client\extensions\java6\auth\oauth2\FileCredentialStore.java:197:87: Expected @throws tag for 'IOException'. [JavadocMethod]
[WARN] D:\a\google-oauth-java-client\google-oauth-java-client\google-oauth-client-java6\src\main\java\com\google\api\client\extensions\java6\auth\oauth2\VerificationCodeReceiver.java:30:34: Expected @throws tag for 'IOException'. [JavadocMethod]
[WARN] D:\a\google-oauth-java-client\google-oauth-java-client\google-oauth-client-java6\src\main\java\com\google\api\client\extensions\java6\auth\oauth2\VerificationCodeReceiver.java:33:31: Expected @throws tag for 'IOException'. [JavadocMethod]
[WARN] D:\a\google-oauth-java-client\google-oauth-java-client\google-oauth-client-java6\src\main\java\com\google\api\client\extensions\java6\auth\oauth2\VerificationCodeReceiver.java:36:22: Expected @throws tag for 'IOException'. [JavadocMethod]
Audit done.
```

**Comment by schrosolver** [comment]
I see that some of the throws clauses have been added already, Can I add for the remaining ones and create a PR?

**Comment by schrosolver** [comment]
Fixed here #743
